src/Microsoft.Windows.ComputeVirtualization/HcsFunctions.cs

```diff
@@ -5,36 +5,91 @@
 
 namespace Microsoft.Windows.ComputeVirtualization
 {
+    internal struct ProcessInfo
```

Comment:
Is this an HCS interop struct? If so can you name it exactly the same as the C++